spec/lib/navigation/state_file_ny_question_navigation_spec.rb

```diff
@@ -1,7 +1,6 @@
 require "rails_helper"
 
 RSpec.describe Navigation::StateFileNyQuestionNavigation do
-
```

Comment:
Are you using an automated formatter? I noticed a lot of lines like this being removed. If so, which one? (Maybe we should incorporate it in our process)

Reply:
i really should get an automated formatter but no i'm doing these manually when i notice them 🙈